proxy/boot/manager-host.js

```diff
@@ -163,7 +163,7 @@ module.exports = function setupHooks(server) {
 
     // TODO(ritch) this is going to block and be incredibly slow...
     boot(client, {
-      appRootDir: path.join(path.dirname(require.resolve('strong-pm')), 'lib', 'client')
+      appRootDir: path.join(path.dirname(require.resolve('strong-mesh-models')), 'client')
```

Comment:
I believe as of v3.x strong-pm is just getting this all from strong-mesh-models anyway, so there's no reason to load it via strong-pm. @kraman is my assumption here correct?